examples/02_first_level_models/plot_bids_features.py

```diff
@@ -143,3 +143,15 @@
 # We can get a latex table from a Pandas Dataframe for display and publication
 from nistats.reporting import get_clusters_table
 print(get_clusters_table(z_map, norm.isf(0.001), 10).to_latex())
+
+#########################################################################
+# Using the computed FirstLevelModel and contrast information,
+# we can quickly create a summary report.
+
+from nistats.reporting import make_glm_report
+
+output_filepath = 'generated_report_flm_bids_features.html'
+report = make_glm_report(model=model,
+                         contrasts='StopSuccess - Go',
+                         )
+report.save_as_html(output_filepath)
```

Comment:
why are you saving it here? maybe we should just make it the output of a jupyter cell and possibly `open_in_browser`? I think it is best not to write anything in the source directory (although some other nistats examples do it)